sci-biology/aghermann/aghermann-1.0.3.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+DESCRIPTION="A sleep-research experiment manager, EDF viewer & Achermann's Process S model runner"
+HOMEPAGE="http://johnhommer.com/academic/code/aghermann"
+SRC_URI="http://johnhommer.com/academic/code/aghermann/source/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="x11-libs/gtk+:3
+	 dev-libs/libconfig
+	 dev-libs/libunique:3
+	 sci-libs/gsl
+	 sci-libs/itpp
+	 sci-libs/fftw
+	 dev-libs/libconfig
+	 media-libs/libsamplerate
+	 x11-libs/vte
+	 dev-lang/lua"
```

Comment:
Only use tab for indention. And when you are at it, please sort the deps alphabetically (that's not required, but I like it ;) )